README.md

```diff
@@ -241,6 +242,10 @@
 
 * [MyNixOS](https://mynixos.com/) - Graphical editor for Nix flakes. Create and manage configurations and modules for NixOS and Nix home-manager. Rather a Nix generator than a Nix editor, because it does not allow to import Nix files.
 
+## NixOS Base Configurations
+
+* [NixOS Config Base](https://github.com/dch82/NixOS-Config-Base) - Basic modularized configuration for NixOS, with support for Home Manager
```

Comment:
The description is ambiguous. I would expect a NixOS configuration with home-manager integration but it looks like it contains separate NixOS and home-manager configurations.

Reply:
Ah, yes... That's because I use home-manager standalone as it is more flexible.

I could change the description or make it integrated if needed.

Tell me if I should do either.

Thanks!

Reply:
I changed the description to be less ambiguous.
Check out latest commit.